tests/Functional/Security/AccessTest.php

```diff
@@ -238,6 +238,67 @@ public function testUserForbiddenField(): void
         $this->assertResponse($query, $expected, static::USER_ADMIN, 'access');
     }
 
+    public function testUserNullableField(): void
+    {
+        $expected = [
+            'data' => [
+                'user' => [
+                    'nullField' => null,
+                ],
+            ],
+        ];
+
+        $query = <<<'EOF'
+query MyQuery {
+  user {
+    nullField
+  }
+}
+EOF;
```

Comment:
```suggestion
        $query = <<<GQL
        query MyQuery {
          user {
            nullField
          }
        }
        GQL;
```
There is no need to align EOF strings to the left side, cause PHP automatically removes unnecessary spaces according to the closing EOF. It also makes sense to use some meaningful text, describing the string, instead of just generic `EOF`. Also the quotes in the `'EOF'` can be removed, if there is no variable interpolation.

Reply:
`<<<'GQL'` we don't have vars here